CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed 
+
+- Replaced `PhoneNumberValidator::$errorNames` with `PhoneNumberValidator::ERROR_NAMES` (BC Break)
```

Comment:
I think this formulation may disturb people upgrading (this will be print in automatic dependency upgrade and make people afraid)

```suggestion
- Remove PHP 8.1 deprecation by replacing `PhoneNumberValidator::$errorNames` with `PhoneNumberValidator::ERROR_NAMES` (minor bc break, if you were using this parameter, use the new const)
```

Something like that sounds better to me.

Reply:
you mean Symfony 6.1 :) updated, thanks